.golangci.yml

```diff
@@ -66,10 +69,6 @@ issues:
   - path: test/pipelinerun_test\.go
     linters:
     - unused
-  - path: pkg/reconciler/pipelinerun/pipelinerun_test\.go
-    # This 11,000+ line file is too big for unused code detection.
-    linters:
-    - unused
```

Comment:
What would happen if add more tests to this file? its about 10,378 line file right now

Reply:
There's obviously going to be _some_ max size that is out of our control -- so my 2c is that we'll cross this bridge when we come to it.